<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Lenovo-PC" name="com.uxian.foodgroup.FoodPostUserInfo.FoodUserVsTest" tests="4" failures="1" timestamp="19 Nov 2015 08:14:31 GMT" time="0.116" errors="0">
  <testcase name="activateCommentTest" time="0.033" classname="com.uxian.foodgroup.FoodPostUserInfo.FoodUserVsTest">
    <failure type="java.lang.AssertionError" message="expected [Success] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [Success] but found [null]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at com.uxian.foodgroup.FoodPostUserInfo.FoodUserVsTest.activateCommentTest(FoodUserVsTest.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- activateCommentTest -->
  <testcase name="noRequiredTest" time="0.030" classname="com.uxian.foodgroup.FoodPostUserInfo.FoodUserVsTest"/>
  <testcase name="noRequiredTest" time="0.034" classname="com.uxian.foodgroup.FoodPostUserInfo.FoodUserVsTest"/>
  <testcase name="noRequiredTest" time="0.019" classname="com.uxian.foodgroup.FoodPostUserInfo.FoodUserVsTest"/>
</testsuite> <!-- com.uxian.foodgroup.FoodPostUserInfo.FoodUserVsTest -->
